### PR TITLE
Add missing node.get_firmware_update_capabilities command

### PR DESCRIPTION
**Pull request by raman325**
This command has been available for a while but we haven't taken advantage of it.

### CODE REVIEW

File: zwave_js_server/model/firmware.py [hunk 1]
```diff
@@ -13,6 +13,67 @@
...
+    def supports_activation(self) -> Optional[bool]:
+        """Return whether node supports delayed activation of the new firmware."""
+        if not self.firmware_upgradable:
+            raise TypeError("Firmware is not upgradeable.")
+        if (val := self.data["supportsActivation"]) == VALUE_UNKNOWN:
+            return None
+        assert isinstance(val, bool)
+        return val
+
+    def to_dict(self) -> Dict[str, Optional[Union[bool, List[int]]]]:
```
Comment (MartinHjelmare) [inline zwave_js_server/model/firmware.py:65]: Let's return a TypedDict here.

File: zwave_js_server/model/firmware.py [hunk 2]
```diff
@@ -13,6 +13,67 @@
...
+    continuesToFunction: Union[bool, str]
+    supportsActivation: Union[bool, str]
+
+
+class FirmwareUpdateCapabilities:
+    """Model for firmware update capabilities."""
+
+    def __init__(self, node: "Node", data: FirmwareUpdateCapabilitiesDataType) -> None:
+        """Initialize class."""
+        self.node = node
```
Comment (MartinHjelmare) [inline zwave_js_server/model/firmware.py:30]: Why do we set the node here? It's not used.

File: zwave_js_server/model/firmware.py [hunk 3]
```diff
@@ -13,6 +13,67 @@
...
+class FirmwareUpdateCapabilitiesDataType(TypedDict, total=False):
+    """Represent a firmware update capabilities dict type."""
+
+    firmwareUpgradable: bool  # required
+    firmwareTargets: List[int]
+    continuesToFunction: Union[bool, str]
+    supportsActivation: Union[bool, str]
+
+
+class FirmwareUpdateCapabilities:
```
Comment (MartinHjelmare) [inline zwave_js_server/model/firmware.py:25]: We could describe the response with a pydantic model directly.

File: zwave_js_server/model/firmware.py (hunk 2 above)
Comment (raman325) [reply zwave_js_server/model/firmware.py:30]: I thought it was needed when I first started but you are right, I'll remove

File: zwave_js_server/model/firmware.py (hunk 3 above)
Comment (raman325) [reply zwave_js_server/model/firmware.py:34]: if we are going to do that should we be looking at that elsewhere as well? I almost feel like it should be done as a single big initiative like we did with the event models, what do you think?

Comment (MartinHjelmare) [reply zwave_js_server/model/firmware.py:34]: We should probably do it in the same release, yes. We don't need to do everything in the same PR, I think.

**Review by MartinHjelmare — APPROVED** [review]

File: zwave_js_server/model/firmware.py (hunk 3 above)
Comment (raman325) [reply zwave_js_server/model/firmware.py:34]: Good point

**Pull request merged**